corehq/apps/accounting/bootstrap/features.py

```diff
@@ -120,4 +120,6 @@
     privileges.ODATA_FEED,
 ]
 
-enterprise_v0 = advanced_v0 + []
+enterprise_v0 = advanced_v0 + [
+    privileges.APP_USER_PROFILES,
```

Comment:
Is this really the first enterprise-only feature? Does the enterprise plan just have higher limits for users, etc.?

@biyeun